tests/test_poster.py

```diff
@@ -14,4 +14,7 @@ def test_locations(self):
         assert (self.poster.locations()["data"] == "testData")
 
     def test_tables(self):
-        assert (self.poster.tables()["data"] == "testTables")
\ No newline at end of file
+        assert (self.poster.tables()["data"] == "testTables")
+
+    def test_customers(self):
```

Comment:
@vladarefiev shouldn't you instruct the mock Poster server to return "testCustomers" for "customers" call? I see it's made so for test_locations and test_tables

Reply:
@amihaiemil good catch! thanks